llvm/test/CodeGen/AArch64/sme2-intrinsics-vdot.ll

```diff
@@ -1,15 +1,14 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sme2,+sme-i16i64 -verify-machineinstrs < %s | FileCheck %s
+; RUN: llc -verify-machineinstrs < %s | FileCheck %s
 
+target triple="aarch64-linux-gnu"
```

Comment:
It is better to keep the triples and attributes as part of the run line, as it makes adding multiple run lines much cleaner.